README.md

```diff
@@ -20,8 +20,8 @@ implemented on top of redlock to manage access to the requested lock. This queue
 Requirements
 ------------
 
- * python 2.7
- * tox
+ * python 2.7 or python 3.4
+ * redlock-py
```

Comment:
I think we should remove this paragraph altogether. Saying we require tox isn't all that useful, given we provide "how to run the tests" below.